src/solver/simulation/sim_calcul_economique.cpp

```diff
@@ -562,14 +562,9 @@ void SIM_RenseignementProblemeHebdo(const Study& study,
             for (uint k = 0; k != linkCount; ++k)
             {
                 auto& lnk = *(studyruntime.areaLink[k]);
-                const int tsIndex = (lnk.directCapacities.width != 1) ? lnk.timeseriesNumbers[0][year] : 0;
 
-                assert((uint)hourInYear < lnk.directCapacities.height);
-                assert((uint)tsIndex < lnk.directCapacities.width);
-                assert((uint)tsIndex < lnk.indirectCapacities.width);
-
-                ntc.ValeurDeNTCOrigineVersExtremite[k] = lnk.directCapacities[tsIndex][hourInYear];
-                ntc.ValeurDeNTCExtremiteVersOrigine[k] = lnk.indirectCapacities[tsIndex][hourInYear];
+                ntc.ValeurDeNTCOrigineVersExtremite[k] = lnk.directCapacities.getCoefficient(year, hourInYear);
+                ntc.ValeurDeNTCExtremiteVersOrigine[k] = lnk.indirectCapacities.getCoefficient(year, hourInYear);
```

Comment:
Performance : 
1. swap loops on links & hours
2. use `TimeSeries::getColumn` a single time instead of `TimeSeries::getCoefficient` 8700 times